<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>766</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>766</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionUDP_Listener"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="menuTools"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="serverSelectionDockWidget">
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Device Selection</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="SeverSelectionWidget" name="serverSelectionWidget"/>
  </widget>
  <widget class="QDockWidget" name="logDockWidget">
   <property name="windowTitle">
    <string>Log Window</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="udpListenerDockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>UDP Listener</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="UdpListener" name="udpListenerWidget"/>
  </widget>
  <widget class="QDockWidget" name="programRecorderDockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Recordings</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="ProgramRecorderWidget" name="programRecorderWidget"/>
  </widget>
  <widget class="QDockWidget" name="programTesterDockWidget">
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>Tester</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="ProgramTesterWidget" name="programTesterWidget">
    <property name="toolTipDuration">
     <number>-1</number>
    </property>
   </widget>
  </widget>
  <action name="actionUDP_Listener">
   <property name="text">
    <string>UDP Listener</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+L</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>SeverSelectionWidget</class>
   <extends>QWidget</extends>
   <header>severselectionwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UdpListener</class>
   <extends>QWidget</extends>
   <header>udplistener.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ProgramRecorderWidget</class>
   <extends>QWidget</extends>
   <header>programrecorderwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <slots>
  <slot>searchDevices()</slot>
 </slots>
</ui>
